op-challenger/flags/flags.go

```diff
@@ -142,6 +147,8 @@ var requiredFlags = []cli.Flag{
 	L1EthRpcFlag,
 	FactoryAddressFlag,
 	DatadirFlag,
+	RollupRpcFlag,
+	L1BeaconFlag,
```

Comment:
nit: if we're placing these in the `requiredFlags` slice, can we move the flag instantiations under the `// Required Flags` comment on line 29 as opposed to under the `// Optional Flags` on line 57? Maybe in the future there's a better way to do this like splitting the instantiation groupings?

Reply:
Yep good catch.